### DISCUSSION TITLE
first header not responsive (lags every 5 seconds)

### DISCUSSION THREAD

**Discussion by swinxx** [docmost/docmost#482] [Help / Questions]
yesterday i installed docmost for my project documentations
it was a bit frustrating to install, but it works now as a docker image on my nas.

when creating a new page the header is responds so slow to my typing, that it is nearly not possible to write a single word without messing it up completely. it probably has to do with the autosave function, which seems to kick in every 5 seconds or so. then the link of the site changes too. when typing in exactly this moment, the text gets screwed up

is there anything i can change in the variables to make this thing a bit more responsive?

as far as i tested it out, this problem is only when typing into the first header.

best wishes.

**Comment by salazar-tech** [comment]
I have the same issue. I thought it was my fault for running it using a cloudflare tunnel but then I noticed that the lag only is present at the header bar.

**Comment by mdkaleel** [comment]
I am also facing the same issue. whenever create a new page , when I type the first header the text , the text is not typed as I typed
![docmostHeaderissue](https://github.com/user-attachments/assets/d122b4f6-2d6f-4bb4-8344-254393546790)

How to fix this?

**Comment by fuscodev** [comment]
I fix it in PR #1154
